packages/main/src/List.ts

```diff
@@ -1040,7 +1040,7 @@ class List extends UI5Element {
 
 	focusFirstSelectedItem() {
 		// only enabled items are focusable
-		const firstSelectedItem = this.getFirstItem(x => x.selected && !x.disabled);
+		const firstSelectedItem = this.getFirstItem(x => x.selected && !x._focusable);
```

Comment:
Should this be kept as not focusable? Might be just a missed conversion to -> x._focusable